shared/js/intl_helper.js

```diff
@@ -0,0 +1,170 @@
+(function(global) {
+  'use strict';
+
+  /*************************
+
+  IntlHelper.define('shortDate', 'datetime', {
+    year: 'numeric',
+    month: 'long',
+  });
+
+  function setValue() {
+    var formatter = IntlHelper.get('shortDate');
+    console.log(formatter.format(new Date()));
+  }
+
+  setValue();
+
+  IntlHelper.observe('shortDate', setValue);
+  
+   ****************************/
+  var objectCache;
+
+  const knownObjects = {
+    'datetime': {
+      set: function(options) {
+        const customOptions = Object.assign({}, options);
```

Comment:
I guess I'm paranoid about memory usage and wanted to keep it undefined unless someone uses it. But then, if someone is loading the helper, they should use it, right? I'll make the change.